apps/src/StudioApp.js

```diff
@@ -2920,4 +2920,28 @@ StudioApp.prototype.showRateLimitAlert = function () {
   });
 };
 
-export const singleton = new StudioApp();
+let instance;
+
+export function singleton() {
+  if (!instance) {
+    instance = new StudioApp();
+  }
+  return instance;
+}
+
+if (IN_UNIT_TEST) {
+  let __oldInstance;
+
+  module.exports.stubStudioApp = function () {
```

Comment:
I think so. stubbing is when you temporarily replace something with something else.